src/VideoStreaming/VideoReceiver.cc

```diff
@@ -872,7 +874,7 @@ VideoReceiver::_updateTimer()
                 stop();
             }
         } else {
-            if(!running() && !_uri.isEmpty() && _videoSettings->streamEnabled()->rawValue().toBool()) {
+            if(!_stop && !running() && !_uri.isEmpty() && _videoSettings->streamEnabled()->rawValue().toBool()) {
```

Comment:
We should call this `_stopped`

Reply:
Theoretically, this variable comes from the user input and not to handle the pipeline, if stop comes after "stopping the video stream" I would agree.